docs/src/main/asciidoc/_templates/template-concept.adoc

```diff
@@ -32,6 +32,50 @@ include::{includes}/extension-status.adoc[]
 - xref:{doc-guides}/doc-concept.adoc#concept[Quarkus documentation content types: Concept guides]
 - xref:{doc-guides}/doc-reference.adoc[Quarkus style and content guidelines]
 
+== Anchors for in-file and cross-file navigation
+
+* To create an anchored ID, use lower-case characters, Separate each word with `,` and enclose the ID in `[[]]` as in the example below.
++
+[source,asciidoc]
+----
+[[title-heading]]
+== Title heading
+----
+
+* To call an anchor created in the same file, insert the anchored ID in a `<<>>` xref macro.
++
+[source,asciidoc]
+----
+<<title-heading>>
+----
+
+* To call an anchored ID from a different file, use the full file name and anchored ID separated by `#`,
+and specify the human-readable URL caption.
++
+[source,asciidoc]
+----
+xref:<other-file-name>.adoc#title-heading[Title heading]
+----
+
+* To create an anchor with a custom URL caption example, specify the anchored ID and desired name separated by `,` without white space.
++
+[source,asciidoc]
+----
+<<title-heading,Title heading description that fits the context of your content>>
+----
++
+WARNING: Do not use the <<>> format with the verbatim heading or section description, such as
+<<Title heading>>.
```

Comment:
i would put this paragraph before the xref one.

Reply:
OK, will do